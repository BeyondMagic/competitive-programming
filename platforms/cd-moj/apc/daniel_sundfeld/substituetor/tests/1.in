O rato roeu a roupa do rei de Roma
rato
esquilo
