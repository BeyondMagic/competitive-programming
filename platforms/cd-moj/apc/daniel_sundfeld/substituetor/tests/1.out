O esquilo roeu a roupa do rei de Roma
